src/game/gameResult.js

```diff
@@ -0,0 +1,18 @@
+import { displayModal } from "./gameOverModal";
+import { stopTime } from "./gameTimer";
+
+export function finishGame() {
+  let settlement;
+  let result;
+  const computerPoints = window.localStorage.getItem("computerPoints");
```

Comment:
Use functions from `utils` that @MaciekPon implemented.

Reply:
done